src/main/java/hw11and12/FileManagerNIO.java

```diff
@@ -0,0 +1,93 @@
+package hw11and12;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class FileManagerNIO {
+    private static final Logger logger = Logger.getLogger(FileManagerIO.class.getName());
```

Comment:
As it is a constant, then the name should be uppercase, do not forget it please, `logger` -> `LOGGER`